libs/network/network.c

```diff
@@ -148,7 +169,9 @@ void networkGetHostByName(
 }
 
 
-
+/**
+ * Create a network.
```

Comment:
You've done it again ;) https://github.com/espruino/Espruino/blob/master/libs/network/network.h#L77

I know the original wasn't Doxygen-formatted, and it is small - but it does provide more info...